charts/prometheus-snmp-exporter/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Prometheus SNMP Exporter
 name: prometheus-snmp-exporter
-version: 3.0.0
+version: 3.0.2
```

I think this should probably be `5.0.0` as it is technically a breaking change as we're changing the default of an existing value. It would likely be transparent to users as you mentioned but if anyone is mirroring images in a private registry or has some security constraints on the registries they access etc then they would need to actively make changes.
```suggestion
version: 5.0.0
```

---

Updated